zebra/zebra_vty.c

```diff
@@ -984,6 +984,44 @@ DEFPY (show_route_table_vrf,
 	return CMD_SUCCESS;
 }
 
+DEFPY (show_route_all_table_vrf,
+       show_route_all_table_vrf_cmd,
+       "show <ip$ipv4|ipv6$ipv6> route all [vrf <NAME$vrf_name|all$vrf_all>] [json$json]",
```

Comment:
I'm not a huge fan of this placement of 'all'.  Vrf selection is typically immediately after the first discrimnator.  `show ip pim vrfx', `show bgp vrf X', 'show ip ospf vrf X'. I'd like it to be consistent

Reply:
what about 'show ip route vrf all tables' ?

Reply:
that sounds fine to me